src/App.scss

```diff
@@ -0,0 +1,97 @@
+$background: #043949;
+$darker-background: #022A39;
+$default-foreground: #0C4A5B;
+$button-color: #ffcd70;
+$dark-font-color: #090B0C;
+$light-font-color: #f0f2f5;
+
+// FONTS
+$fontType: Helvetica;
+$fontTypeBackup: "Source Sans Pro";
+
+$fontLight: 300;
+$fontNormal: 400;
+$fontBold: 600;
+
+$titleSizeMobile: 24px;
+$subtitleSizeMobile: 20px;
+$buttonSizeMobile: 20px;
+$bodySizeMobile: 14px;
+$secondarySizeMobile: 10px;
+
+$titleSizeTablet: 24px;
+$subtitleSizeTablet: 20px;
+$buttonSizeTablet: 20px;
+$bodySizeTablet: 14px;
+$secondarySizeTablet: 10px;
+
+$titleSizeDesktop: 32px;
+$subtitleSizeDesktop: 24px;
+$buttonSizeDesktop: 24px;
+$bodySizeDesktop: 16px;
+$secondarySizeDesktope: 12px;
+
+// SPACING
+$guttersMobile: 24px;
+$cardSeparationMobile: 24px;
+$sectionSpacingMobile: 40px;
+$cardPaddingMobile: 24px;
+
+$guttersTablet: 32px;
+$cardSeparationTablet: 32px;
+$sectionSpacingTablet: 48px;
+$cardPaddingTablet: 24px;
+
+$guttersDesktop: 48px;
+$cardSeparationDesktop: 48px;
+$sectionSpacingDesktop: 80px;
+$cardPaddingDesktop: 24px;
+
+// BUTTON SPECS
+$buttonRadius: 10px;
+
+#root {
+  max-width: 1280px;
+  margin: 0 auto;
+  padding: 2rem;
+  text-align: center;
+}
+
+// .logo {
+//   height: 6em;
+//   padding: 1.5em;
+//   will-change: filter;
+//   transition: filter 300ms;
+// }
+
+// .logo:hover {
+//   filter: drop-shadow(0 0 2em #646cffaa);
+// }
+
+// .logo.react:hover {
+//   filter: drop-shadow(0 0 2em #61dafbaa);
+// }
+
+// @keyframes logo-spin {
+//   from {
+//     transform: rotate(0deg);
+//   }
+
+//   to {
+//     transform: rotate(360deg);
+//   }
+// }
+
+// @media (prefers-reduced-motion: no-preference) {
+//   a:nth-of-type(2) .logo {
+//     animation: logo-spin infinite 20s linear;
+//   }
+// }
+
+// .card {
+//   padding: 2em;
+// }
+
+// .read-the-docs {
+//   color: #888;
+// }
```

Comment:
Standard practice is to have exactly one empty line at the end of files, if you don't have one then GitHub gives you this cheerful ⛔ sign to let you know!